Cargo.lock

```diff
@@ -2869,7 +2914,7 @@ version = "0.3.26"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0c6b5c64445ba8094a6ab0c3cd2ad323e07171012d9c98b0b15651daf1787a10"
 dependencies = [
- "clap",
+ "clap 2.34.0",
```

Comment:
We should track down whatever's using this in the other fuel repos and update those crates to also use clap 3 to avoid dependency bloat, but it's probably fine to do this in a follow-up PR.

Reply:
Cool, looks like the `fuel-gql-client` repo is still relying on `structopt`. I'll make a separate PR.